README.md

```diff
@@ -1,4 +1,4 @@
-# Logbook
+## Logbook: HTTP request and response logging

```

Comment:
The former web developer in me cringes, because there should be a single h1 headline on a page. This feels wrong. Have you tried:

```html
<h1>Logbook <small>HTTP request and response logging</small></h1>
```